app.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "Kutt",
+    "description": "Free Modern URL Shortener.",
+    "website": "https://kutt.it",
+    "repository": "https://github.com/thedevs-network/kutt",
+    "logo": "https://kutt.it/images/logo.svg",
+    "success_url": "/",
+    "env": {
+      "NPM_CONFIG_PRODUCTION": {
+        "value": "false"
+      },
+      "PGSSLMODE": {
+        "value": "no-verify"
```

Comment:
https://devcenter.heroku.com/articles/connecting-heroku-postgres#connecting-in-node-js